crates/consensus/src/config.rs

```diff
@@ -1,99 +1,8 @@
 //! Reth block execution/validation configuration and constants
-use reth_executor::{Config as ExecutorConfig, SpecUpgrades};
-use reth_primitives::{BlockNumber, U256};
-
-#[cfg(feature = "serde")]
-use serde::{Deserialize, Serialize};
 
 /// Initial base fee as defined in: https://eips.ethereum.org/EIPS/eip-1559
 pub const EIP1559_INITIAL_BASE_FEE: u64 = 1_000_000_000;
 /// Base fee max change denominator as defined in: https://eips.ethereum.org/EIPS/eip-1559
 pub const EIP1559_BASE_FEE_MAX_CHANGE_DENOMINATOR: u64 = 8;
 /// Elasticity multiplier as defined in: https://eips.ethereum.org/EIPS/eip-1559
 pub const EIP1559_ELASTICITY_MULTIPLIER: u64 = 2;
```

Comment:
We should rename this module to `constants` since it no longer has anything to do with config